meetings/2023-08-16.md

```diff
@@ -43,20 +51,117 @@
+* eP: For me it would be interesting to see which are actually product classes. Can you highlight those, composing multiple product classes in single implementation seems common. I think we still do not have a clear view on how that should be done.
+* SC: But this is a subtopic. Not sure if it necessarily resolves it.
+* WT: This was more concrete issue, I think current definition of Server is very broad. I don't think is wrong but for conformance class should use something like Resource Server.
+* SC: ... realises the more general issue: https://github.com/solid/specification/issues/480
+* SC: Can we move this till after Wouter's proposal?
+* eP: I think Resource Server is a great example to start from. It's already used in a few reports, and we also use Solid Storage a lot. Solid Notifications is not directly depend on the Solid spec, but we assume that solid storage will implement notifications resource server.
+
+ACTION: WT posts in discussion
 
 ### Return ETag headers on PUT requests
-https://github.com/CommunitySolidServer/CommunitySolidServer/issues/632
+URL: https://github.com/CommunitySolidServer/CommunitySolidServer/issues/632
 
 * SC: Let's have additional eyes/minds on this. Cases in which the ETag header can(not) be used in `PUT` and `PATCH` responses. Potentially introduce a requirement or add advisory in Solid Protocol.
+* SC: General issue is that CSS might want more input on how it should approach. I raise this here so that people can have a look, both at CSS and Solid Protocol, to possibly refine the spec.
 * WT: What is the added value for the Solid ecosystem, on top of the affordances already provided by HTTP?
+* SC: It's about whether the interpretation of the spec is correct? Is it within what the Solid protocol allows? Or is it required?
+* WT: Given that current solid spec doesn't require anything about it, it is implementation choice, what would be the advantage of mandating it by the spec?
+* SC: I thought ETag was required.
+* TBL: For the logic to work we need ETag to work. It provides conditional writes. (explains Etag...)
+* SC: Solid says it is a MUST. Clients can expect ETag.
+* TBL: Clients need to rely on it. Is this issue on whether it is also required on put (unless the ETag RFC already requires it).
+* SC: Solid protocol does not distinguish between the method; it relies on the RFC. The question is what the behavior is on PUT and PATCH.
+* AC: I want to make sure that any decision we make is consistent with current HTTP spec and how current browsers work. Etags work on representations. If a server returns a 204 we do not want the browser to cache that. We also don't want it to conflict with HTTP: it is really focused on representations.
+* WT: Wanted to join AC on that about representations. What I meant earlier what is not in the current protocol, for GET, ETag is clearly there. Same could go for PUT, based on representation, it can return an ETag.
```

Comment:
```suggestion
* AC: I want to make sure that any decision we make is consistent with current HTTP spec and how current browsers work. Etags work on representations. If a server returns a 204, we do not want the browser to cache that. We also don't want it to conflict with HTTP; it is really focused on representations.
* WT: Wanted to join AC on that about representations. What I meant earlier, what is not in the current protocol; for GET, ETag is clearly there. Same could go for PUT; based on representation, it can return an ETag.
```